HCPE-995 - Add issue lifecycle documentation and issue templates (#93)
hashicorp/terraform-provider-hcp: .github/ISSUE_TEMPLATE/bug-report.md

---
name: 🐛 Bug Report
about: If something isn't working as expected 🤔.
labels: bug
---

<!---

> Please note the following potential times when an issue might be in Terraform core:

Reviewer: ```suggestion
Please note there are times when an issue might be in Terraform core:
```

### Terraform Configuration Files

<!--- Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation. --->

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Debug Output

<!---
Please provide a link to a GitHub Gist containing the complete debug output. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

To obtain the debug output, see the [Terraform documentation on debugging](https://www.terraform.io/docs/internals/debugging.html).
--->

### Panic Output

<!--- If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`. --->

### Steps to Reproduce

<!--- Please list the steps required to reproduce the issue. --->

1. `terraform apply`

### Expected Behavior

<!--- What should have happened? --->

### Actual Behavior

<!--- What actually happened? --->

### Important Factoids

> <!--- Are there anything atypical about your accounts that we should know? --->

Reviewer: ```suggestion
<!--- Is there anything atypical about your accounts that we should know? --->
```
